kivy/uix/checkbox.py

```diff
@@ -172,6 +173,22 @@ def on_active(self, instance, value):
         if self._check_variable:
             self._check_variable = False
 
+    def on_group(self, *largs):
+        groups = ToggleButtonBehavior._ToggleButtonBehavior__groups
+        if self._previous_group:
+            group = groups[self._previous_group]
+            for item in group[:]:
+                if item() is self:
+                    group.remove(item)
+                    break
+        group = self._previous_group = self.group
+        if group not in groups:
+            groups[group] = []
+        r = ref(self, ToggleButtonBehavior._clear_groups)
+        groups[group].append(r) 
+        if self.active:
+            self._release_group(self)
```

Comment:
Instead of copying the code from button's `on_group`, it's better to just call `super`. and then do stuff after it.

Reply:
You're perfectly right. Fixed it.